tests/e2e/scripts/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:20.04
+
+RUN apt-get update && \
+    apt-get install -y curl openvpn bridge-utils iputils-ping && \
```

Comment:
Having the packages in alphabetical order improves maintainability since it makes it easier for humans to avoid future duplicates.